.github/workflows/daily-builds.yml

```diff
@@ -107,7 +107,8 @@ jobs:
           echo "*************************** START BUILDING **************************************"
           CC=gcc-12 CXX=g++-12 cmake .. -DCMAKE_BUILD_TYPE=Debug -GNinja -DWITH_UNWIND=OFF \
             -DCMAKE_CXX_FLAGS="-Wl,-ld_classic" \
-            -DCMAKE_C_COMPILER="gcc-12" -DCMAKE_CXX_COMPILER="g++-12"
+            -DCMAKE_C_COMPILER="gcc-12" -DCMAKE_CXX_COMPILER="g++-12" \
+            -DWITH_TIERING=OFF
```

Comment:
I think it would've been better if we automatically set WITH_TIERING=OFF in cmake if we see that it's macos

Reply:
oh nice! +1 since we can check the os in cmake :tada: